lib/ansible/modules/cloud/misc/helm_cli.py

```diff
@@ -0,0 +1,595 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: helm_cli
+short_description: Manages Kubernetes packages with the Helm package manager
+description:
+  - Install, upgrade, delete packages with the Helm package manager.
+version_added: "2.10"
+author:
+  - Lucas Boisserie (@LucasBoisserie)
+  - Matthieu Diehr (@d-matt)
+requirements:
+  - "helm (https://github.com/helm/helm/releases)"
+  - "yaml (https://pypi.org/project/PyYAML/)"
+options:
+  binary_path:
+    description:
+      - The path of a helm binary to use, relative to the 'service_path'
+        unless you supply an absolute path.
+    required: false
+    type: path
+  chart_ref:
+    description:
+      - chart_reference on chart repository
+      - path to a packaged chart
+      - path to an unpacked chart directory
+      - absolute URL
+      - Required when release_state is set to present
+    required: false
+    type: str
```

Comment:
Maybe use
```suggestion
    type: path
```
instead? Then Ansible will replace things like `~` IIRC.